src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -0,0 +1,100 @@
+#include "ExecuteProcess.hpp"
+#include "Addresses.hpp"
+#include "App.hpp"
+#include "Hook.hpp"
+#include "Systems/ScriptSystem.hpp"
+#include "stdafx.hpp"
+
+namespace
+{
+bool isAttached = false;
+
+struct FixedWString
+{
+    uint32_t length;
+    uint32_t maxLength;
+    wchar_t* str;
+};
+
+bool _Global_ExecuteProcess(void* a1, RED4ext::CString* aCommand, FixedWString* aArgs,
+                            RED4ext::CString* aCurrentDirectory, char a5);
+Hook<decltype(&_Global_ExecuteProcess)> Global_ExecuteProcess(Addresses::Global_ExecuteProcess,
+                                                              &_Global_ExecuteProcess);
+
+bool _Global_ExecuteProcess(void* a1, RED4ext::CString* aCommand, FixedWString* aArgs,
+                            RED4ext::CString* aCurrentDirectory, char a5)
+{
+    if (strstr(aCommand->c_str(), "scc.exe") == nullptr)
+    {
+        return Global_ExecuteProcess(a1, aCommand, aArgs, aCurrentDirectory, a5);
+    }
+    std::wstring original = aArgs->str;
+    std::wstring buffer;
```

Comment:
Use fmt buffer, I would prefer this over appending strings. Then we can get the string view and pass it along.

Reply:
With `fmt::wmemory_buffer`, like this?
```cpp
    auto buffer = fmt::wmemory_buffer();
    if (scriptSystem->IsUsingRedmod())
    {
        spdlog::info("Using RedMod configuration");
        format_to(std::back_inserter(buffer), scriptSystem->GetRedModArgs());
    }
    else
    {
        format_to(std::back_inserter(buffer), aArgs.str);
    }
    format_to(std::back_inserter(buffer), LR"( -compilePathsFile "{}")", scriptSystem->CreatePathsFile());
```

Reply:
Yes!